### PR TITLE
Admin management add delete edit events

### PR DESCRIPTION
**Pull request by holotsvan**
[https://github.com/ita-social-projects/GreenCity/issues/4554](https://github.com/ita-social-projects/GreenCity/issues/4554)
[https://github.com/ita-social-projects/GreenCity/issues/4557](https://github.com/ita-social-projects/GreenCity/issues/4557)
[https://github.com/ita-social-projects/GreenCity/issues/4558](https://github.com/ita-social-projects/GreenCity/issues/4558)

- Functionality to add, edit, delete events
- Add element to upload images
- Element for confirmation modal
- Page for creating/editing
- Add messages to messages.properties
- Delete messages_ru.properties

### CODE REVIEW

File: dao/.attach_pid12916
Comment (Maryna-511750) [inline dao/.attach_pid12916:1]: Are these files needed here?

**Comment by HelenSotnik** [comment]
Agree with previous comment , 4 empty files looks like should not be included to PR and 2 new issues could be fixed.

**Review by HelenSotnik — APPROVED** [review]

**Review by Maryna-511750 — APPROVED** [review]

**Review by IliaWithHat — APPROVED** [review]
